rt/features/logging/src/main/java/org/apache/cxf/ext/logging/AbstractLoggingInterceptor.java

```diff
@@ -73,6 +78,14 @@ public long getInMemThreshold() {
         return threshold;
     }
 
+    public void addSensitiveElementNames(final List<String> sensitiveElementNames) {
+        maskSensitiveHelper.addSensitiveElementNames(sensitiveElementNames);
+    }
+
+    public void addSensitiveHeaders(final List<String> sensitiveHeaders) {
```

Comment:
Maybe rename to HTTPHeaders to make the meaning clear (as opposed to say SOAP headers)

Reply:
ProtocolHeaders indeed make more sense. Other question if it makes sense to support masking sensitive SOAP headers as well?